packages/components/src/toggle-group-control/toggle-group-control-option-base/component.tsx

```diff
@@ -3,7 +3,7 @@
  */
 import type { ForwardedRef } from 'react';
 // eslint-disable-next-line no-restricted-imports
```

Comment:
We should be able to remove this `ESlint` comment?

Reply:
Good observation! Actually that comment is correct, because we normally folks shouldn't be able to import directly from ariakit (they should use the library through the `@wordpress/components` package) — or at least, this was the deal with reakit, and I believe that at least for now, we should apply to ariakit too.

I went ahead and updated the ESLint config to disallow ariakit imports, and consequently added more "disable" comments where needed in the components package.